e2e/support/helpers/e2e-ui-elements-helpers.js

```diff
@@ -8,7 +8,7 @@ export function popover() {
 }
 
 export function mantinePopover() {
-  const MANTINE_POPOVER = "[data-popover=mantine-popover]";
+  const MANTINE_POPOVER = "[data-element-id=mantine-popover]";
```

Comment:
This makes sure the data attribute name is consistent with the rest of the code. We currently use `data-element-id` to set ID to elements that we used to track with CSS class but with the CSS module migration. We don't actually return any classes in tests, that's why `data-element-id` is needed.